Parsers/ASimAuditEvent/Parsers/vimAuditEventBarracudaWAF.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: Audit Event ASIM parser for Barracuda WAF
```

Comment:
(1) Warning: Empty value in 150 records (100.0%)  in mandatory field [EventEndTime] (Schema:AuditEvent)
(1) Warning: Empty value in 150 records (100.0%)  in mandatory field [EventStartTime] (Schema:AuditEvent)

Reply:
AS data is not present in workspace just closing this comment